ansible/README.md

```diff
@@ -11,8 +11,7 @@ ansible-galaxy install geerlingguy.pip
 
 Generate configuration files:
 ```bash
-$ cd ansible
-$ ./generate_config.sh
+$ ./ansible/generate_config.sh
 # follow the instructions
```

Comment:
I don't think this is sufficient, like  I can see in the script that there are two questions, I guess. 
What's the nickname of your server?
I could reason that this was the same name as the `.yml`, in this case `netcup`
But the next question
What will be the new domain?
I can see that it is used on line 12, `traefik_domain="traefik.${domain}`, but what should I choose there? 
I see on link 34, you have given an example `traefik.tactile.news`, should I choose that for the test?

Reply:
the domain is user-specific, so we cannot put it in the README, mine is `100eyes.roschaefer.de`

Reply:
cool, sorry, I hadn't understood that I could skip to the part https://github.com/tactilenews/100eyes/blob/cfe988dfb79065b4f155facd28b00984b80ccffa/ansible/README.md#installation